spec/schemas/ml._common.yaml

```diff
@@ -187,6 +187,160 @@ components:
         framework_type:
           type: string
           description: The framework type.
+    PredictResponse:
+      type: object
+      properties:
+        inference_results:
+          # Update this once the PR for ML Model APIs (#733) has been merged
+          type: array
+          items:
+            $ref: '#/components/schemas/InferenceResults'
+        status:
+          type: string
+          description: The status.
```

Comment:
Is it possible to define the status as an enum? If not, that's fine; you can leave it as it is.

Example here: https://github.com/opensearch-project/opensearch-api-specification/blob/edf9d6f0dbf1068345b58cb9e2c3464fa9839226/spec/schemas/replication._common.yaml#L30

Reply:
Yes, I think we can define status as an enum here.